src/messages/MMDSBeacon.h

```diff
@@ -200,6 +200,7 @@ class MMDSBeacon final : public PaxosServiceMessage {
 
   static constexpr int HEAD_VERSION = 8;
   static constexpr int COMPAT_VERSION = 6;
+  static constexpr int PRIORITY = CEPH_MSG_PRIO_HIGH;
```

Comment:
```suggestion
  static constexpr int HIGH_PRIO = CEPH_MSG_PRIO_HIGH;
  ```

Reply:
I wouldn't rename it. This is a field in the MMDSBeacon class, so it should state the priority selected for that kind of message. The initializer is already clear that the beacon's priority should be high.

Reply:
Your reasoning is correct, and I'd +1 this if the code was somewhere in a generic method where multiple different messages would receive their priorities, like this:

```
void setup_messaging() {
  // ...
  client_message.set_priority(DEFAULT);
  beacon_message.set_priority(HIGH);
  //...
}
```

However, in this case, we are _inside_ the Beacon class. And, as part of its initialization, we configure its priority on the message. You should read it as

```
BeaconMessage::init() {
  set_priority(MY_PRIORITY);
}
```

and if you are interested, you can check for 

```
 BeaconMessage {
   MY_PRIORITY = CEPH_MSG_PRIO_HIGH;
...
```

Reply:
However, unless we are using `BeaconMessage::PRIORITY` somewhere, I'd agree with the general note that 

```
  set_priority(CEPH_MSG_PRIO_HIGH);
```

looks good :)

Reply:
I'm just going to drop this commit. It's not worth the trouble.